motion/core_ext/object/duplicable.rb

```diff
@@ -78,6 +78,6 @@ def duplicable?
 
 class BigDecimal
   def duplicable?
-    true
+    false
```

Comment:
This looks like something that changed in Ruby 1.9.3 -> 2.0.0 after some quick checks

RubyMotion:
```bash
(nil)? BigDecimal.new('4.56').dup
2015-11-14 21:58:09.047 MotionSupport[33557:13510978] can't copy Float (TypeError)
=> #<TypeError: can't copy Float>
```

Ruby 1.9.3:
```
1.9.3-p551 :001 > BigDecimal.new('4.56').dup
=> TypeError: can't copy BigDecimal
```

Ruby 2.0.0:
```
2.0.0-p647 :002 > BigDecimal.new('4.56').dup
 => #<BigDecimal:7fc657219920,'0.456E1',18(18)>
```

Ruby 2.2.2:
```
2.2.2 :001 > BigDecimal.new('4.56').dup
=> #<BigDecimal:7fee3632b1b8,'0.456E1',18(18)>
```